public/components/stub-modal/stub-modal.html

```diff
@@ -41,7 +41,7 @@ <h4 class="modal-title">{{ modalTitle }}</h4>
                     </select>
                 </div>
             </div>
-            <div class="form-group pull-left col-xs-8" ng-if="mode !== 'import' && contentName !== 'Atom'">
+            <div class="form-group pull-left col-xs-8" ng-if="contentName !== 'Atom' && !['Atom Workshop', 'Media Atom Maker'].includes(importHandler.name)">
```

Comment:
We don't want to show the commissioning section when we're importing / creating an atom.

Reply:
By section, I mean section of the form (including Commissioning Desk and Commissioned Length). Rather than the 'section' field.